tide_news.install

```diff
@@ -130,7 +130,7 @@ function tide_news_update_8003(&$sandbox) {
     $sandbox['#finished'] = $count ? 0 : 1;
   }
 
-  $batch_size = 10;
+  $batch_size = 100;
```

Comment:
Altering deployed update hooks is a bad idea.

Because this change is going to be Premier specific you could create a patch using GitHub and reference it in the root composer.

Reply:
Yep @GROwen , I won't merge this, I opened this to create the .diff to include in the project.